thesis-de/content-de.tex

```diff
@@ -1868,6 +1842,32 @@ \subsection{Vorbereitung der Evaluierung}
   nur die Weboberfläche zugreift, sondern auch die Datenerfassung, um Daten
   abzuspeichern, kann es hier zu massiven Performance-Problemen kommen, die
   durch die Weboberfläche verursacht werden können.
+  \1 Plugin-Erweiterung. Das Frontend muss eine Plugin-Erweiterung
+  unterstützen. Es dient dazu, dass im späteren Verlauf, wenn das Produkt
+  bereits beim Kunden eingesetzt wird, auf bestimmte Kundenwünsche direkt
+  eingegangen werden kann. Diese können dann mit schon zur Verfügung gestellten
+  Plugins zum Beispiel in der Community oder aber durch selbst programmierte
+  Plugins realisiert werden. Die Plugins werden in Javascript geschrieben. Die
```

Comment:
Die Plugins werden in Javascript geschrieben -> das ist eine Anforderung?
Kann das Plugin nicht in eine andere Sprache entwickelt werden? Wieso nicht?